apis/validator/beacon_committee_subscriptions.yaml

```diff
@@ -22,6 +22,9 @@ post:
             properties:
               committee_index:
                 $ref: '../../beacon-node-oapi.yaml#/components/schemas/Uint64'
+              committees_at_slot:
+                $ref: '../../beacon-node-oapi.yaml#/components/schemas/Uint64'
+                description: "Number of committees at the returned slot"
```

Comment:
even though it's valid openapi spec, it won't display description.
should be:
```
committees_at_slot:
                allOf:
                    - $ref: '../../beacon-node-oapi.yaml#/components/schemas/Uint64'
                    - description: "Number of committees at the returned slot"
```

Reply:
This should be fixed now, thanks